### PR TITLE
feat: add presentation strategies

### PR DESCRIPTION
**Pull request by JasonLantz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,11 @@ List of Elements with their current respective design version:
 | Inputs | v1.0.0 |
 | Buttons | v1.0.0 |
 | Calendar | v1.0.0-beta.1 |
+| Grid | 1.0.0 |
+| Templates | 1.0.0 |
+| Presentation Strategies | 1.0.0 |
```
Comment (StommePoes) [inline README.md]: everyone else has a v in front.

File: docs/elements/grid.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+The grid consists of four key parts: the Container, Container Margins, Columns, and Gutters.
+
+All columns reside within the Container, which is located at the root level of a page. Depending on the breakpoint, the container will have Container Margins of various sizes. The container also has a maximum width property, beyond which only the container margins grow in size.
+
+The container is divided into 12 proportional Columns which contain actual content. These are separated by Gutters of a fixed size (which varies based on the breakpoint again).
+
+See the redlines for the exact dimensions of each feature at each breakpoint.
+
+
+## Rediline Details
```
Comment (StommePoes) [inline docs/elements/grid.md]: Also a link here before or after text description?

File: docs/elements/templates.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+- Full 12 column width at all breakpoints
+
+Responsive:
+- Maintain full 12 column width
+
+#### Two Column - (right)
+{{#demo}}
+  <div class='pe-template__double' style='background:magenta;'>
+    <div class='pe-template__double--main'>
+      <h6 style='background:cyan;text-align:center;height:40px;'>Content Area</h6>
```
Comment (StommePoes) [inline docs/elements/templates.md]: Why h6 here? What you've got in the grid etc is perfect. Random headings seem random. Or were you going for something here?

File: docs/elements/templates.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  </div>
+{{/demo}}
+
+#### Two Column - (left)
+{{#demo}}
+  <div class='pe-template__double' style='background:magenta;'>
+    <div class="pe-template__double--sidebar">
+      <h6 style='background:cyan;text-align:center;height:40px;'>Sidebar Area</h6>
+    </div>
+    <div class='pe-template__double--main'>
```
Comment (StommePoes) [inline docs/elements/templates.md]: I think you're the first one using BEMmy class names in any sensible way :P

File: docs/elements/templates.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+{{/demo}}
+
+
+## Shared Features
+Spacing:
+- Directly adjoining templates have no vertical space between them
+- Small Gap is 40px total
+- Large Gap is 80px total
+
+Backgrounds & Borders:
```
Comment (StommePoes) [inline docs/elements/templates.md]: & amp?

File: scss/_variables.scss [hunk 6]
```diff
@@ -53,13 +53,70 @@ $pe-colors: (
...
   sm: (min: 480px,  max: 767px),
   md: (min: 768px,  max: 1023px),
   lg: (min: 1024px, max: 1279px),
   xl: (min: 1280px)
 ) !default;
 
+//Grid
+//
+
+$pe-columns             : 12 !default;
```
Comment (StommePoes) [inline scss/_variables.scss]: Not saying you should change this, but my poor retarded eyeballs have trouble tracking large distances like that.

File: scss/mixins/_grid.scss [hunk 7]
```diff
@@ -0,0 +1,85 @@
+//Grid Mixins
+
+@mixin generate-container($pe-container-max-width, $pe-responsive-breakpoints) {
+  .pe-container {
+    position      : relative;
```
Comment (StommePoes) [inline scss/mixins/_grid.scss]: Can this go away or are we position-ing  children inside?

File: scss/_variables.scss (hunk 6 above)
Comment (JasonLantz) [inline scss/_variables.scss]: will ∆ to  _ : _  (one space separator)

File: README.md (hunk 1 above)
Comment (JasonLantz) [inline README.md]: will add the v to the last 3

File: docs/elements/grid.md (hunk 2 above)
Comment (JasonLantz) [inline docs/elements/grid.md]: adding a link to the design page...

File: docs/elements/templates.md (hunk 3 above)
Comment (JasonLantz) [inline docs/elements/templates.md]: i was looking for something without padding that i could use inside the div...

File: docs/elements/templates.md (hunk 5 above)
Comment (JasonLantz) [inline docs/elements/templates.md]: & to and....

File: scss/mixins/_grid.scss (hunk 7 above)
Comment (JasonLantz) [inline scss/mixins/_grid.scss]: removing relative position on container...

File: docs/elements/templates.md (hunk 3 above)
Comment (StommePoes) [inline docs/elements/templates.md]: divs, p's, headings, none have default padding but Elements might be adding some somewhere. I thought we're only putting a bottom margin on p's though. 

Ultimately might not matter, I'm keeping copy-pasta-eaters in mind. If you don't find another easy sub, leave it.

**Comment by wyseguyonline** [comment]
Aside from Mallory's comments...the rest LGTM

**Comment by StommePoes** [comment]
woozles and pips

**Comment by wyseguyonline** [comment]
test hipchat comment

**Comment by udhadpa** [comment]
All tests are passed and reviewed. Good to merge.

**Pull request merged**